doc/rfd/rfd/0009/README.md

```diff
@@ -0,0 +1,266 @@
+or other means of deployment.
+
+Definitions of *must*, *should*, *required*, etc, are as explained in
+[rfc2119](https://tools.ietf.org/html/rfc2119).
+
+Some behaviour has been purposely left vague. When something is not concretely
+specified, those implementing it have the freedom to implement it as they desire
+(e.g: as of v0.1 path of the folder structure is not defined, so it can be
+anywhere in the repo or resulting artifacts).
+
+The interface consists in targets that can be called and perform their defined
+action, together with helper files and examples of configuration for the charts.
+
+Those targets and files must be in the following structure of folders and executables:
+
+```
+├── uninstall/*
+├── clobber/*
+├── chart/*
+├── configure/*
+├── install/*
+├── wait/*
+├── include/*
+│
+├── one-offs/*
+│
+├── include/*
+├── example-values/*
+│
+└── spec_ver
+```
+
+This structure as a whole may be published together with the Helm chart. This
+can be for example as artifacts of a chart release, or present in the chart
+repository, but it is recommended that these files are bundled inside of the chart:
+this simplifies consumption downstream.
+
+Each target is required to be a folder (E.g: `install/`) containing executable
+files starting with digits (E.g: `10_do-foo.sh`, `20_do-bar.rb`). Targets
+defined in the `one-offs/` folder also follow this structure.
+
+Upon target call, all the executable files starting with digits will get
+executed in alphanumeric order.
+It is optional for the folder to contain other files/folders besides those
+starting with digits. Those would not be automatically executed upon target
+call (e.g: readme files, local include folders, etc).
+
+Each of these targets:
+
+- Must assume that the deployment/testing env is already loaded, with access to
+  kube, cf, helm operations.
+- Must return retcode `0` on success, `!= 0` if otherwise.
+- Must consume and create files on `pwd` only.
+- Must be idempotent: running it will give you the same result. E.g: calling
+  `configure` several times with the same outputs gives you the same
+  `values.yaml` as result. Calling `install` with the same `values.yaml` gives
+  you the same deployment.
+- Must be isolated: E.g; implementation of `install` does not call on `configure`
+  itself.
+  They may, though, call shared implementation code in the `include` folder of
+  their shared root path.
+- Chart values must be only consumed from yaml files, and not using `--set`.
+- When useful, should save resulting artifacts of execution into
+  `$pwd/artifacts`. E.g: `$pwd/artifacts/cats-run.log`
+
+It is not recommended for targets to take arguments and options on execution
+besides the expected default behaviour defined here.
+  
+Deployment and automation scripts must expose the following targets, with the
+defined behaviour:
+
+- `chart`: Obtains the latest public release of the chart at hand, and puts it
```

Comment:
We no longer need to call this `chart`; it can just be `download` at this point, since nothing else depends on this being a chart.